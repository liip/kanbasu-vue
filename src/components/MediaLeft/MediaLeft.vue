<template>
  <div class="media__left">
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: 'MediaLeft',
};
</script>
