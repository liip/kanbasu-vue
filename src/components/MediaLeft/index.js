import MediaLeft from './MediaLeft.vue';

MediaLeft.install = function(Vue) {
  Vue.component(MediaLeft.name, MediaLeft);
};

export default MediaLeft;
