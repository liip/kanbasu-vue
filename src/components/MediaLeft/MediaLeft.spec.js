import { shallowMount } from '@vue/test-utils';
import MediaLeft from './MediaLeft.vue';

describe('MediaLeft.vue', () => {
  it('renders', () => {
    const wrapper = shallowMount(MediaLeft, {
      slots: {
        default: 'slot',
      },
    });
    expect(wrapper).toMatchSnapshot();
  });
});
