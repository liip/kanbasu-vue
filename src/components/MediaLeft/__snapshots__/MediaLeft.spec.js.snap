// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaLeft.vue renders 1`] = `<div class="media__left">slot</div>`;
